gems/nokogiri/CVE-2021-3537.yml

```diff
@@ -0,0 +1,34 @@
+---
+gem: nokogiri
+cve: 2021-3537
+ghsa: 286v-pcf5-25rc
+url: https://nokogiri.org/CHANGELOG.html#security_8
```

Comment:
Use the permalink: https://nokogiri.org/CHANGELOG.html#1114-2021-05-14

Reply:
Done